src/devices/Is31fl3730/Registers.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Register addresses for the Function Register.
+    /// </summary>
+    public static class FunctionRegister
```

Comment:
There's probably no need to make this public.

Reply:
Good call. Fixed.